vendor/github.com/docker/go-plugins-helpers/volume/api.go

```diff
@@ -120,3 +134,27 @@ func (h *Handler) handle(name string, actionCall actionHandler) {
 		sdk.EncodeResponse(w, res, res.Err)
 	})
 }
+
+func (h *Handler) handleMount(name string, actionCall mountActionHandler) {
+	h.HandleFunc(name, func(w http.ResponseWriter, r *http.Request) {
+		var req MountRequest
+		if err := sdk.DecodeRequest(w, r, &req); err != nil {
```

Comment:
I don't think we should be swallowing the error here. It should be returned to the user no?